src/lib/supabase/library/mockData.ts

```diff
@@ -0,0 +1,105 @@
+import {v4 as uuid} from 'uuid';
```

Comment:
maybe rename this to "randomUUID" so it's clear that it's mock data
uninstall this `uuid` package and use the built in node uuid generator from their module `crypto
```suggestion
const {randomUUID} = require('crypto');
```